website/docs/cli/commands/index.mdx

```diff
@@ -57,7 +57,7 @@ All other commands:
 Global options (use these before the subcommand, if any):
   -chdir=DIR    Switch to a different working directory before executing the
                 given subcommand.
-  -help         Show this help output, or the help for a specified subcommand.
+  -help         Show this help output or the help for a specified subcommand.
```

Comment:
I don't think we can change this because it is a representation of the actual CLI output, not just docs about it. We could correct the CLI output itself, but I don't want to update the docs without doing that first or they won't match.